src/backend/base/langflow/utils/constants.py

```
@@ -9,12 +9,27 @@
     "gpt-4o",
+    "gpt-4o-mini",
     "gpt-4-turbo-preview",
     "gpt-4-0125-preview",
     "gpt-4-1106-preview",
     "gpt-4-vision-preview",
     "gpt-3.5-turbo-0125",
     "gpt-3.5-turbo-1106",
+    "gpt-4.1",
+    "gpt-4.1-mini",
+    "gpt-4.1-nano",
+    "gpt-4.5", 
```

Ruff (0.11.9)

Trailing whitespace

Remove trailing whitespace

(W291)